src/test/java/core/basesyntax/service/ReportServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import static junit.framework.TestCase.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static final String TITLE_FOR_RESULT = "fruit,quantity";
+    private static final String NEW_LINE = System.lineSeparator();
```

Comment:
don't need to create a constant for System.lineSeparator(), it's obvious by name of the method, what this line for